### ISSUE TITLE
[BUG] Notification not showing

### ISSUE THREAD

**Issue by veldrict** [jhomlala/alice#96]
**Describe the bug**
My project use alice for check network using dio. 
After i add package onesignal_flutter, notification not showing anymore. It work fine on android but on iOS is stop showing.

**To Reproduce**
1. install onesignal_flutter latest version
2. set up onesignal_flutter
3. Get data from api
4. Notification not showing

**Expected behavior**
Notification show like on android

**Flutter doctor**
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.5.2, on macOS 11.6 20G165 darwin-x64, locale
    en-ID)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.61.2)
[!] Connected device
    ! No devices available
```

**Alice version**
- Version: 0.2.4

**Smartphone :**
 - Device: iPhone 7 and iphone emulator
 - OS: iOS 15

**Additional context**
i think the problem is from local notification on alice.

**Comment by jhomlala** [comment]
Seems like flutter_local_notifications plugin doesn't work well with onesignal_flutter. Please check this issue: https://github.com/OneSignal/OneSignal-Flutter-SDK/issues/197#issuecomment-590153195 . You can check if this solution will work for you.

**Comment by veldrict** [comment]
hmm, maybe you can use awesome notification for exchange the local notification. 
because someone recommend it in this comment[OneSignal/OneSignal-Flutter-SDK#197 (comment).](https://github.com/OneSignal/OneSignal-Flutter-SDK/issues/197#issuecomment-953761551).
And I cant change the OneSignal because it from my client.

**Comment by chandrabezzo** [comment]
For logging networking with toggleable UI, please try [analytics_debugger](https://pub.dev/packages/analytics_debugger)

**Comment by areebmomin** [comment]
I tried adding the below line of code in the **_AppDelegate.swift_** file in the **_iOS_** module. It's working fine now.

```
if #available(iOS 10.0, *) {
      UNUserNotificationCenter.current().delegate = self as? UNUserNotificationCenterDelegate
}
```
